src/components/Products/ProductsListItem.vue

```diff
@@ -0,0 +1,97 @@
+<template>
+  <div class="product-list-item">
+    <div
+      class="product-list-item__picture"
+      v-bind:style="{ backgroundImage: 'url(' + imgUrl + ')' }"
+    ></div>
+    <div class="product-list-item__description">
+      <span class="product-list-item__description__title" v-if="name.length < 20">{{ name }} </span>
```

Comment:
raz masz po jednym propie w linii, raz wszystkie w linii. Trzeba to ujednolicić. Najlepiej po jednym propie na linie